source/tutorial/triggers-fts.txt

```diff
@@ -1,8 +1,8 @@
 .. _triggers-fts-tutorial:
 
-====================================================
-Build an Application Using Triggers and Atlas Search
-====================================================
+==================================================
+Build a Percolator Using Triggers and Atlas Search
```

Comment:
Opting for a one line title ("Build Reverse Search into your Application") since we mention Triggers and Atlas Search in the first line - let me know if this works for you!